frontend/src/lib/components/general/Header.svelte

```diff
@@ -43,18 +43,12 @@
 						icon={mdiCompare}
 						on:click={() => goto(`${getProjectRouteFromURL($page.url)}/compare`)}
 					/>
-					{#if $project}
-						{#await ZenoService.getProjectStats($project.uuid) then stats}
-							{#if $project.editor || stats.numCharts > 0}
-								<HeaderIcon
-									pageName={'chart'}
-									tooltipContent={'Create charts from your slices and metrics.'}
-									icon={mdiChartBoxOutline}
-									on:click={() => goto(`${getProjectRouteFromURL($page.url)}/chart`)}
-								/>
-							{/if}
-						{/await}
-					{/if}
+					<HeaderIcon
+						pageName={'chart'}
+						tooltipContent={'Create charts from your slices and metrics.'}
+						icon={mdiChartBoxOutline}
+						on:click={() => goto(`${getProjectRouteFromURL($page.url)}/chart`)}
+					/>
```

Comment:
I only showed this for non-editors when there actually were charts. Otherwise they are brought to an empty page. Hence the numCharts check. Do we always want to show this? If we do, what do we show if there are no charts?

Reply:
It was flickering every time on the sidebar when any interaction happened b/c of the fetch and it was bothering me aha. Maybe we can just put text in the tab if there are no charts.

Reply:
Sounds good to me. Just wanted to bring this up since seeing a blank page might be weird.